src/multiple_shooting_method.jl

```diff
@@ -0,0 +1,74 @@
+export MultipleShootingObjective, multiple_shooting_method
+
+immutable MultipleShootingObjective{F,C} <: Function
+  multiple_shooting_cost::F
+  constraints::C
+end
+
+my_problem_new_parameters = function (prob::ODEProblem,p,t1,t2,data)
+  f = (t,u,du) -> prob.f(t,u,p,du)
+  uEltype = eltype(p)
+  tspan = (uEltype(t1),uEltype(t2))
+  u0 = [uEltype(data[:,t1][i]) for i in 1:length(prob.u0)]
+  ODEProblem(prob.f,u0,(t1,t2))
+end
```

Comment:
Only works for ODEs but this can get cleaned up later. Should open an issue on this.